lib/server.js

```diff
@@ -40,18 +42,21 @@ export default function (fastify, { filepath, lazy = false }, done) {
     if (!reader) {
       reader = new Reader(filepath)
     }
-    return reader.getStyle(fastify.listeningOrigin)
+    const baseUrl = new URL(fastify.prefix, fastify.listeningOrigin)
```

Comment:
is `fastify.prefix` always a string? if it's null or undefined at any point, i think this will result in undesired behavior

```js
console.log(new URL(undefined, 'https://localhost:8080').href) // logs 'https://localhost:8080/undefined'
```

Reply:
based on TS types, seems like it's always a string. Now it's just a matter of trust 😄